NOTICE.txt

```diff
@@ -8,6 +8,14 @@ This document includes a list of open source components used in Mattermost Web A
 
 --------
 
+## @types/highlight
+
+---
+
+## @typescript-eslint/parser
```

Comment:
Was added via https://github.com/mattermost/mattermost-webapp/commit/fef8d8d2778e383b7750d5247a513e3ea03cd0a0#diff-b9cfc7f2cdf78a7f4b91a753d10865a2.

Reply:
Chris if you can provide links for these and then I can fill in the details.

Reply:
@amyblais Go to https://www.npmjs.com/ and search for the package names.